packages/react-migration-v8-v9/src/components/Theme/v9ThemeShim.ts

```diff
@@ -104,7 +104,9 @@ const mapAliasColors = (palette: IPalette, inverted: boolean): ColorTokens => {
     colorNeutralBackgroundInvertedDisabled: whiteAlpha[10],
     colorNeutralStencil1: palette.neutralLight,
     colorNeutralStencil2: palette.neutralLighterAlt,
-    colorBackgroundOverlay: blackAlpha[10],
+    colorNeutralStencil1Alpha: inverted ? whiteAlpha[10] : blackAlpha[10],
+    colorNeutralStencil2Alpha: inverted ? whiteAlpha[10] : blackAlpha[5],
```

Comment:
According to PR description

```suggestion
    colorNeutralStencil2Alpha: inverted ? whiteAlpha[5] : blackAlpha[5],
```